core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -151,45 +151,43 @@ var TopicEditorPage = function() {
   };
```

Comment:
Can we wait for elements to be visible before trying to drag and drop them above?